projects/core/src/util/http-params-uri.encoder.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { HttpParamsURIEncoder } from './http-params-uri.encoder';
+
+describe('HttpParamsURIEncoder', () => {
+  const httpParamsURIEncoder = new HttpParamsURIEncoder();
```

Comment:
For simplicity sake, rename this to just `encoder` - this will imply the importance of the variable. Similarly to `service` variable in services tests.